vnext/PropertySheets/ReactPatches.targets

```diff
@@ -5,12 +5,19 @@
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <Target Name="EvaluatePatchReactInputsOutputs" BeforeTargets="PrepareForBuild">
+  <PropertyGroup>
+    <PatchTargetIncluded>true</PatchTargetIncluded>
+    <ReactPackageCommon>$(ReactNativePackageDir)\ReactCommon\</ReactPackageCommon>
+    <ReactCommon>$(ReactNativeDir)\ReactCommon\</ReactCommon>
+    <DeforkingPatchesCommon>$(ReactNativeWindowsDir)\DeforkingPatches\ReactCommon\</DeforkingPatchesCommon>
+  </PropertyGroup>
+
+  <Target Name="EvaluatePatchReactInputsOutputs" BeforeTargets="PrepareForBuild;PoisonUnpatchedHeaders">
```

Comment:
nit: I should rename this target and its items. These were named like this when they did something slightly different, but I never updated naming when I changed their purpose.